tasks/build.py

```diff
@@ -654,6 +662,7 @@ def submit_job(job, cfg):
         build_env_cfg[config.BUILDENV_SETTING_SLURM_PARAMS],
         time_limit,
         job.slurm_opts,
+        f"--job-name='{job_name}'",
```

Comment:
Also here, only use `--job-name` is the job_name is not `None`?

Reply:
Addressed in https://github.com/EESSI/eessi-bot-software-layer/pull/273/commits/c614a96f801ae2c79f9f1fa9cd5d3cdecbc156db